pkg/gatherers/clusterconfig/gather_monitoring_pvs.go

```diff
@@ -0,0 +1,135 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	coreV1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"sigs.k8s.io/yaml"
+)
+
+// GatherMonitoringPVs Collects Persistent Volumes from openshift-monitoring namespace
+// which matches with ConfigMap configuration yaml
+//
+// ### API Reference
+// - https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/configmap.go
+// - https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/persistentvolume.go
+//
+// ### Sample data
+// - docs/insights-archive-sample/config/persistentvolumes/monitoring-persistent-volume.json
+//
+// ### Location in archive
+// - `config/persistentvolumes/{persistent_volume_name}.json`
+//
+// ### Config ID
+// `clusterconfig/monitoring_persistent_volumes`
+//
+// ### Released version
+// - 4.14
+//
+// ### Changes
+// None
+func (g *Gatherer) GatherMonitoringPVs(ctx context.Context) ([]record.Record, []error) {
+	kubeClient, err := kubernetes.NewForConfig(g.gatherProtoKubeConfig)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	mg := MonitoringPVGatherer{client: kubeClient.CoreV1()}
+
+	name, err := mg.getDefaultPrometheusName(ctx)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	return mg.gather(ctx, name)
+}
+
+type MonitoringPVGatherer struct {
+	client coreV1.CoreV1Interface
+}
+
+// getDefaultPrometheusName returns prometheus name as it's described on the configmap
+// or an error collection from the attempts to retrieve that information
+func (mg MonitoringPVGatherer) getDefaultPrometheusName(ctx context.Context) (string, error) {
```

Comment:
Nit: I think the naming here is a little bit confusing. It's not getting any Prometheus name IMO, but rather VolumeClaim name used for (or in) Prometheus configuration.